### PR TITLE
Added new feature fixed, src

### PR DESCRIPTION
**Pull request by dhieru**
Added new src directory, moved index.ts file in src directory
modified tsconfig.json file
Please review @basant001@gmail.com

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "strict": true,
-        "outDir": "bin",
```
Comment (dhieru) [inline tsconfig.json:4]: LGTM, next time run npm run format

**Review by dhieru — COMMENTED** [review]
See comments

File: tsconfig.json (hunk 1 above)
Comment (dhieru) [reply tsconfig.json:4]: ok sounds good Thanks

**Review by dhieru — COMMENTED** [review]
@basant001@gmail.com please approve this Pull request

**Review by basant001 — APPROVED** [review]

**Review by basant001 — APPROVED** [review]
Everything looks good.

**Pull request merged**